src/Model/ValueObject/Calendar/DateRange.php

```diff
@@ -40,4 +41,12 @@ public function compare(DateRange $dateRange): int
 
         return 0;
     }
+
+    public function toArray(): array
```

Comment:
Nit: not sure if we need to expose it like this

Reply:
How would you prefer it? I can make a normalizer if you prefer, but I thought that would be so small, and probably never re-used that I opted for this approach.

Reply:
Either a new normalizer or inline.